rules_bot.py

```diff
@@ -25,6 +25,89 @@
 
 self_chat_id = '@'  # Updated in main()
 
+def convertformatting(sending_message : str, entities):
```

Comment:
What need is there for this?

Reply:
It's not needed afterall.. I didn't know there are better options like caption_html or text_html.